api/tests/test_allocation_api.py

```diff
@@ -24,16 +24,24 @@ def test_non_authenticated_user_view_assets(self):
     @patch("api.authentication.auth.verify_id_token")
     def test_get_allocations(self, mock_verify_id_token):
         """Test post new allocation"""
+
         mock_verify_id_token.return_value = {"email": self.other_user.email}
-        response = client.get(
+
+        data = {"asset": self.asset.id, "current_owner": self.asset_assignee.id}
+        client.post(
```

Comment:
As written it is for `getting allocations` 
I added the post since the response had no allocations whatsoever and I couldn't assert that the response had the assigner
Hadn't looked at the docstring since I was adjusting the test, an oversight on my end
Will correct that, thanks

Reply:
You can create the allocation history here with AllocationHistory.objects.create() rather than making a post.